webserver/views/api/v1/test/test_dataset_eval.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+from webserver.testing import ServerTestCase
+from db.testing import TEST_DATA_PATH
+import db.exceptions
+import db.dataset
+import db.dataset_eval
+import db.api_key
+import datetime
+from utils import dataset_validator
+
+import json
+import mock
+import os
+
+class APIDatasetEvaluationViewsTestCase(ServerTestCase):
+
+    def setUp(self):
+        super(APIDatasetEvaluationViewsTestCase, self).setUp()
+
+        self.test_user_mb_name = 'tester'
+        self.test_user_id = db.user.create(self.test_user_mb_name)
+        self.test_user = db.user.get(self.test_user_id)
+
+
```

Comment:
I followed what was done in the other test files. Also the standards say that two lines may be used between sets of different types of functions. I think two lines here are justified because this is a setup function and other functions will be the actual testing functions and hence might be termed as different types of functions.